packages/shared/components/Notification/Notification.tsx

```diff
@@ -118,7 +124,7 @@ function getRenderedContent(
 
   if (typeof content === 'string') {
     return (
-      <Flex justifyContent="space-between" width="100%">
+      <Flex alignItems="center" justifyContent="space-between" width="100%">
```

Comment:
Ideally we'd align the icons to the top with `alignItems="baseline"`. However, either I'm using our icons wrong or [the icons in themselves are not aligned to the baseline](https://blog.prototypr.io/align-svg-icons-to-text-and-say-goodbye-to-font-icons-d44b3d7b26b4#6e0c), that is the bottom of the icon gets aligned to the baseline rather than the icon's center falling on the baseline.

This is how replacing both `alignItems="center"` with `"baseline"` looks like:

![baseline](https://user-images.githubusercontent.com/27113/210258498-112d1e69-3e7c-40c1-af78-132cf394cff7.png)

I even tried refactoring the component so that it's just three elements within a flex but this didn't help because of the fundamental issue with the icons.

Reply:
Yeah, it seems like the icons are not properly aligned :/